pyproject.toml

```
@@ -103,3 +103,11 @@ convention = "google"
 [tool.ruff.lint.isort]
 case-sensitive = true
 lines-after-imports = 2
+
+[tool.mypy]
+python_version = "3.10"
```

What happens if we use the same type specifier as the `requires-python` field above?
```suggestion
python_version = ">=3.10"
```